libraries/AP_Math/crc.cpp

```diff
@@ -374,6 +374,53 @@ uint16_t crc16_ccitt(const uint8_t *buf, uint32_t len, uint16_t crc)
     return crc;
 }
 
+/* right shift table */
```

Comment:
@jfbblue0922,

Great, thanks for this.  We should actually move this to a separate commit prefixed with "AP_Math:".

Reply:
@rmackay9 

thank you.
I have never split commits.
Would it be possible to revert the fix with the 'git revert' command and then make another commit prefixed with AP_Math and AP_RangeFinder?

Reply:
@jfbblue0922,

If you're using Linux/Ubuntu then type "git gui" into the terminal and then a screen like below should pop-up.  "Amend Last Commit", then uncheck the two AP_Math changes, then press Commit again.  Then create a new commit with just the AP_Math changes by staging and then committing them.  Then do a forced push ("git push -f").
![image](https://github.com/ArduPilot/ardupilot/assets/1498098/3635ddc4-7527-4a4e-97be-b1ea4a19fb7e)

If you need my help after this though I can do it for you.